test/boto_test.py

```diff
@@ -135,16 +135,49 @@ def test_get_boto3_without_args(self, mock_get_stats, mock_get_logger, mock_boto
 
 class BotoTest(unittest.TestCase):
 
-    def test_cli_region(self):
+    def test_cli_empty_region(self):
         """
-        --boto-region arguments sets boto.cli_region variable correctly
+        check region if environment not set
         """
-        region = 'us-west-2'
-        self.boto = Boto(
-            region=region,
-        )
+        region = 'us-west-1'
+        with patch.dict('krux_boto.boto.os.environ', clear=True):
+            self.boto = Boto()
+            # Check the region is correctly default set in the mocked os.environ dictionary
+            self.assertEqual(self.boto.cli_region, 'us-east-1')
+
+            self.boto = Boto(
+                region=region
+            )
+            #Check region can be set with command line argument with no environment variables
+            self.assertEqual(self.boto.cli_region, region)
```

Comment:
I split it up into 2 tests. The second one is to test that the command line input still works even if AWS_REGION is not set by the user. I can delete it if it is unnecessary